UPGRADE-1.10.md

```diff
@@ -3,3 +3,9 @@
 ### New API
 
 1. API CartShippingMethod key `cost` has been changed to `price`.
+
+2. Command's `ApiBundle/Command/RegisterShopUser.php` constructor parameters has changed from 
+   `string $firstName, string $lastName, string $email, string $password, ?string $phoneNumber`
+    to
+   `string $firstName, string $lastName, string $email, string $password, bool $subscribedToNewsletter, ?string $phoneNumber`
+    with default `$subscribedToNewsletter` parameter set to `false`.
```

Comment:
Wdyt about changing the order of arguments to have `bool $subscribedToNewsletter = false` as the last one to avoid this note and BC Break? /cc @lchrusciel

Reply:
Done 🚀